App.tsx

```diff
@@ -41,6 +45,7 @@ export default function App() {
         <Stack.Navigator
           initialRouteName={isLoggedIn ? 'HomeScreen' : 'WelcomeScreen'}
         >
+          {/*<Stack.Navigator initialRouteName={'SelectSignUpTypeScreen'}>*/}
```

Comment:
Commented out line

Reply:
Looked right over that one, Thanks for pointing it out!